client/src/pages/Profile/form.ts

```diff
@@ -0,0 +1,25 @@
+import {object, string} from 'yup';
+
+export const defaultNameValue = {
+    newName: '',
+};
+
+export const defaultPasswordValue = {
+    password: '',
+    newPassword: '',
+    passwordConfirmation: '',
+};
+
+export const nameValidation = object().shape({
+    newName: string().required(),
+});
+
+export const passwordValidation = object().shape({
+    password: string().required(),
+    newPassword: string().required(),
+    passwordConfirmation: string()

```

Comment:
create common yup validation for passwordConfirmation, create yupValidationUtils and import validation method